chore: cleanup. (#4527)
timescale/docs: api/uuid-functions/index.md

# UUIDv7 functions <Tag type="community">Community</Tag>

UUIDv7 is a time-ordered UUID that includes a Unix timestamp with millisecond precision in its first 48 bits. Like 
other UUIDs it uses 6 bits for version and variant info. The remaining 74 bits are random.

![UUIDv7 microseconds](https://assets.timescale.com/docs/images/uuidv7-structure-microseconds.svg)

UUIDv7 is ideal anywhere you create lots of records over time. Advantages are:

- **No extra column required to partition by time with sortability**: you can sort UUIDv7 instances by their value. This 
    is useful for ordering records by creation time without the need for a separate timestamp column.

> - **Indexing performance**: UUIDv7s increase with time, so new rows are append near the end of a B-tree. This results in 

Reviewer: ```suggestion
- **Indexing performance**: UUIDv7s increase with time, so new rows are appended near the end of a B-tree. This results in 
```

WHERE uuid_timestamp(e.event_id) >= ref.t0 - INTERVAL '2 days';
```

> Using UUIDv7 means that chunks are excludes at startup, the the query time is reduced to 550ms:

Reviewer: ```suggestion
Using UUIDv7 means that chunks are excluded at startup, and the query time is reduced to 550 ms:
```

WHERE e.event_id >= to_uuidv7_boundary(ref.t0 - INTERVAL '2 days')
```

You use UUIDvs for events, orders, messages, uploads, runs, jobs, spans, and more.
    
## Examples

- **High-rate event logs for observability and metrics**: 

   UUIDv7 gives you globally unique IDs for traceability and time windows such as “last hour”, without the need for a 

>    separate `created_at` column. UUIDv7 create less churn because inserts land at the end of the index, and you can 

Reviewer: ```suggestion
   separate `created_at` column. UUIDv7 creates less churn because inserts land at the end of the index, and you can 
```
